dev-support/Jenkinsfile

```diff
@@ -30,8 +30,9 @@ def publishJUnitResults() {
     boolean surefireReportsExist = findCmdExitCode == 0
     if (surefireReportsExist) {
         echo "XML files found under surefire-reports, running junit"
+        SRC = "${SOURCEDIR}/**/target/surefire-reports/*.xml".replace("$WORKSPACE/","")
```

Comment:
Could you please add a comment here mentioning that the path must be relative to the checkout dir?

Reply:
This path used to have `env.SOURCEDIR` prefix before I made this change - https://github.com/apache/hadoop/blob/ef5dbc7283a9283bc446511c3a49304ad3923037/dev-support/Jenkinsfile#L145. Wasn't `env.SOURCEDIR` resolving to the absolute path back then?

Reply:
I don't think so:
https://github.com/apache/hadoop/blob/ef5dbc7283a9283bc446511c3a49304ad3923037/dev-support/Jenkinsfile#L32

Reply:
Now:
https://github.com/apache/hadoop/blob/trunk/dev-support/Jenkinsfile#L259